tests/sample_qc/test_de_novo.py

```diff
@@ -7,11 +7,60 @@
     calculate_de_novo_post_prob,
     default_get_de_novo_expr,
 )
+from gnomad.utils.annotations import get_copy_state_by_sex
 
 
 class TestDeNovoMutation:
     """Test suite for de novo mutation functions."""
 
+    @pytest.mark.parametrize(
+        "locus, is_xx, expected_diploid, expected_hemi_x, expected_hemi_y",
+        [
+            (
+                hl.locus("chr1", 100000, reference_genome="GRCh38"),
+                True,
+                True,
+                False,
+                False,
+            ),
+            (
+                hl.locus("chrX", 2781479, reference_genome="GRCh38"),
+                False,
+                True,
+                False,
+                False,
+            ),
+            (
+                hl.locus("chrX", 3000000, reference_genome="GRCh38"),
+                False,
+                False,
+                True,
+                False,
+            ),
+            (
+                hl.locus("chrY", 10000000, reference_genome="GRCh38"),
+                False,
+                False,
+                False,
+                True,
+            ),
+        ],
+    )
+    def test_get_copy_state_by_sex(
```

Comment:
this test should be in another file because the function is in a separate file, right? or is this (keeping related functions together) the convention